### PR TITLE
Fix a spec which could fail in some cases

### PR DESCRIPTION
**Pull request by allenwq**

### CODE REVIEW

File: spec/features/system/admin/instance_management_spec.rb [hunk 1]
```diff
@@ -48,8 +48,10 @@
       end
 
       scenario 'I can see all instances' do
-        instances = create_list(:instance, 2)
-        visit admin_instances_path(page: Instance.page.total_pages)
+        create_list(:instance, 2)
+        last_page = Instance.page.total_pages
+        instances = Instance.order_by_id.page(last_page)
```
Comment (allenwq) [inline spec/features/system/admin/instance_management_spec.rb]: @lowjoel I noticed that there is an `order_by_id` in the controller, so I also added it here. But why we need it ? I think the default order is by id?

Comment (lowjoel) [inline spec/features/system/admin/instance_management_spec.rb]: not for postgres, only for mysql

Comment (lowjoel) [inline spec/features/system/admin/instance_management_spec.rb]: this spec is a bit different now. previously: create 2 instances, check that they appear on the page. now: create 2 instances, check that the last page appears... is that sufficient?

Comment (allenwq) [inline spec/features/system/admin/instance_management_spec.rb]: If the order is by id, the 2 instances you created will be in the `instances` variable, so they are actually tested and checked.  Do you have any other idea for testing this ?

Comment (lowjoel) [inline spec/features/system/admin/instance_management_spec.rb]: except when they are split on two pages... maybe we just check that the last page of instances contains at least one of the instances created?

File: spec/features/system/admin/instance_management_spec.rb [hunk 2]
```diff
@@ -49,7 +49,11 @@
 
       scenario 'I can see all instances' do
         instances = create_list(:instance, 2)
-        visit admin_instances_path(page: Instance.page.total_pages)
+        last_page = Instance.page.total_pages
+        instances &= Instance.order_by_id.page(last_page)
+        expect(instances).not_to be_empty
```
Comment (allenwq) [inline spec/features/system/admin/instance_management_spec.rb]: @lowjoel This will guarantee at least one instance we created appears in the last page.

Comment (lowjoel) [inline spec/features/system/admin/instance_management_spec.rb]: :+1:

**Pull request merged**